lib/OpenQA/Utils.pm

```diff
@@ -417,6 +419,12 @@ sub href_to_bugref {
     return $text;
 }
 
+sub url_to_href {
+    my ($text) = @_;
+    $text =~ s(($RE{URI}))(<a href="$1">$1</a>)gx;
+    return $text;
+}
```

Comment:
Don't you want to share code with lib/OpenQA/Schema/Result/Comments.pm without duplicating?